source/settings/plugin/acl-plugin.rst

```diff
@@ -35,30 +36,52 @@ Settings
      file in each mailbox (or :dovecot_core:ref:`mail_control_path`) directory.
      This is the default.
 
-   This backend has the following options:
 
-   ============== ============================================================
-   Name           Description
-   ============== ============================================================
-   <global_path>  If a path is defined, this is the location of the global ACL
-                  configuration file.
-   cache_secs     The interval, in seconds, for running stat() on the ACL file
-                  to check for changes. DEFAULT: 30 seconds
-   ============== ============================================================
+.. dovecot_plugin:setting:: acl
+   :plugin: acl
+   :added: 3.0.0
+   :values: @named_filter_list
 
-   Example:
+   Specifies an ACL entry on global, namespace or mailbox level.
+
+   Has two settings, match and permission. The match setting is same as
+   the identifier in ACL and permission is the permission to grant, or deny, for this user.
```

Comment:
You should add acl_match and acl_permission settings as separate entries as well, and link these to them. And I think try to write sentence in a way that it doesn't sound like they are sub-settings. Like maybe:

Each acl filter must have two settings inside them: acl_match and acl_permission.

Reply:
hm, but the settings are called id and permission, not acl_id and acl_permission?

Reply:
Actually it's acl_id and acl_rights, already in your code. The acl_ prefix is just optional inside acl { .. }. But I think we should generally refer to them using their full names.